tests/scripts/functional_tests.sh

```diff
@@ -88,6 +89,9 @@ function functional_test() {
 
 	# perform the basic api tests
 	basic_api_tests > >(tee "${RESULTS}/basic_api_tests.log") 2>&1
+	
+	# Modify existing autotuneconfig yamls and check for API results
+	modify_autotune_config_tests > >(tee "${RESULTS}/modify_autotune_config_tests.log") 2>&1
```

Comment:
Shouldn't this be `tee -a` and for the command above as well ?

Reply:
It need not be. Because the every time result directory will be different and we want to have different log files for the different test suite